### DISCUSSION TITLE
missing debuffs?

### DISCUSSION THREAD

**Discussion by spamwax** [rgd87/Aptechka#81] [General]
I have tried beta-1 and beta-2 releases and the debuf icons in dungeons don't show up anymore, like Mistveil Bite in Mist, or the poison debuffs that mobs put on you (after 2nd boss in Mist).
It seemed this happened after the fix you made to blacklisted spells and dispel widget

**Comment by d87** [comment]
Yeah, i just noticed yesterday debuffs on betas were broken.
Fixed here https://github.com/rgd87/Aptechka/releases/tag/9.1-beta3
